plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -6,12 +6,11 @@ class Release < ActiveRecord::Base
 
     belongs_to :user
     belongs_to :build
+    belongs_to :project
```

Comment:
and maybe a `before_validation` filter that `project_id ||= build.try(:project_id)`